02 - Community Expansions/02.01 Plugins by Category/Third-Party Integration Plugins.md

```diff
@@ -13,56 +13,56 @@ Plugins to integrate with other services or applications.
 
 ## Plugins in this category
 
-- [[todoist-sync-plugin|Todoist Sync Plugin]]
-- [[obsidian-discordrpc|Discord Rich Presence]]
-- [[obsidian-youglish-plugin|Youglish Plugin]]
-- [[flashcards-obsidian|Flashcards]]
-- [[obsidian-to-anki-plugin|Obsidian_to_Anki]]
-- [[obsidian-apple-reminders-plugin|Apple Reminders]]
-- [[obsidian-imgur-plugin|Imgur Plugin]]
-- [[things-logbook|Things Logbook]]
-- [[notetweet|NoteTweet]]
-- [[obsidian-leaflet-plugin|Obsidian Leaflet]]
-- [[DEVONlink-obsidian|DEVONlink - Open or reveal notes in DEVONthink]]
-- [[imdone-obsidian-plugin|Open cards in imdone from obsidian.]]
-- [[obsidian-languagetool-plugin|LanguageTool Integration]]
-- [[obsidian-readwise|Readwise Community]]
-- [[obsidian-kindle-plugin|Kindle Highlights]]
-- [[beeminder-word-count-plugin|Beeminder Word Count Plugin]]
-- [[obsidian-image-auto-upload-plugin|Image auto upload Plugin]]
-- [[readwise-mirror|Readwise Mirror]]
-- [[obsidian-gist|Gist]]
-- [[obsidian-pandoc|Obsidian Pandoc]]
-- [[obsidian-amazingmarvin-plugin|Amazing Marvin]]
-- [[obsidian-map-view|Map View]]
-- [[open-vscode|Open vault in VSCode]]
-- [[phone-to-roam-to-obsidian|Phone to Roam to Obsidian]]
-- [[obsidian-wordnet-plugin|Obsidian42 - WordNet Dictionary]]
-- [[readwise-official|Readwise Official]]
-- [[obsidian-toggl-integration|Toggl Track Integration]]
-- [[ObsidianAnkiSync|Obsidian Anki Sync]]
-- [[mochi-cards-exporter|Mochi Cards Exporter]]
-- [[obsidian-trello|Obsidian Trello]]
-- [[obsidian-annotator|Annotator]]
-- [[obsidian-hackernews|HackerNews]]
-- [[obsidian-pocket|Pocket integration]]
-- [[obsimian-exporter|Obsimian Exporter]]
-- [[metadata-extractor|Metadata Extractor]]
-- [[uri-commands|URI Commands]]
-- [[obsidian-wikipedia|Wikipedia]]
-- [[obsidian-vale|Vale]]
-- [[obsidian-hypothesis-plugin|Hypothes.is]]
-- [[marginnote-companion|MarginNote Companion]]
-- [[obsidian-habitica-integration|Habitica Sync]]
-- [[obsidian-webhooks|Obsidian Webhooks]]
-- [[obsidian-tweet-to-markdown|Tweet to Markdown]]
-- [[netwik|Netwik]]
-- [[obsidian-jupyter|Jupyter plugin]]
-- [[obsidian-plotly|Plotly]]
-- [[obsidian-ankibridge|AnkiBridge]]
-- [[obsidian-tressel|Tressel Sync for Obsidian]]
-- [[obsidian-graphviz|Obsidian Graphviz]]
-- [[obsidian-power-search|Power Search]]
+- [[todoist-sync-plugin|Todoist Sync Plugin]]: Materialize Todoist tasks within Obsidian notes.
+- [[obsidian-discordrpc|Discord Rich Presence]]: Update your Discord Status to show your friends what you are working on in Obsidian. With Discord Rich Presence.
+- [[obsidian-youglish-plugin|Youglish Plugin]]: Use YouTube to improve your pronunciation. YouGlish gives you fast, unbiased answers about how words is spoken by real people and in context.
+- [[flashcards-obsidian|Flashcards]]: Anki integration
+- [[obsidian-to-anki-plugin|Obsidian_to_Anki]]: This is an Anki integration plugin! Designed for efficient bulk exporting.
+- [[obsidian-apple-reminders-plugin|Apple Reminders]]: 
```

Comment:
This is fine. Another one where there was no description in the linked plugin note.